zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -414,6 +419,28 @@
       commitParagraph($scope.paragraph.title, $scope.paragraph.text, newConfig, newParams);
     };
 
+    var openEditorAndCloseTable = function() {
```

Comment:
The code below from L422 to L443 look very alike, what do you think about making one function that will handle both case?

I have in mind something like
```
var openEditorAndCloseTable = function() {
  manageEditorAndTableState(false, true);
}

var closeEditorAndOpenTable = function() {
  manageEditorAndTableState(true, false);
}

// Please dont get attached to the name :) put whatever you like :D
var manageEditorAndTableState(showEditor, showTable) {
  var newParams = angular.copy($scope.paragraph.settings.params);
  var newConfig = angular.copy($scope.paragraph.config);

  newConfig.editorHide = showEditor;
  newConfig.tableHide = showTable;

  commitParagraph($scope.paragraph.title, $scope.paragraph.text, newConfig, newParams);
}
```
The idea here is to keep this login in one single place so it will help for the future maintainability.